nautobot/ui/src/components/Retrieve/RenderTabs.js

```diff
@@ -12,8 +19,8 @@ function RenderTabs({
     isPluginView,
 }) {
     return (
-        <Tabs>
-            <TabList pl="md">
+        <Tabs variant="outline">
+            <TabList marginX={negateCssVar("md")} paddingX="md">
```

Comment:
Can you offer a short explanation for `negateCssVar`?

Reply:
If raw values were used, negation would be as simple as adding a `-` sign in front of a number (e.g. `-20px`). However, in case of space tokens (which resolve to CSS custom properties), such as `xs`, `sm`, `md`, etc., the negation needs to be somewhat more sophisticated. So what `negateCssVar` does is, it takes space token and multiplies it by `-1` with CSS `calc` function. To put it simple, you can think of `negateCssVar("md")` as something like `-md` or `md * -1`. In reality it is `calc(var(--chakra-space-md), -1)`.